docs/9-roadmap.md

```diff
@@ -4,11 +4,40 @@ sidebar_position: 9
 
 # Roadmap
 
-The project roadmap for Zarf is hosted on a [GitHub Project Board](https://github.com/orgs/defenseunicorns/projects/1) that tracks the issues the Zarf team is working along with future work we are prioritizing.
+## Issue Tracking
+
+The issue board for Zarf is hosted on a [GitHub Project Board](https://github.com/orgs/defenseunicorns/projects/1) that tracks the issues the Zarf team is working along with future work we are prioritizing.
 
 If you would like to add bug reports or feature requests, please [add an issue](https://github.com/defenseunicorns/zarf/issues) to the GitHub repository under the appropriate template.  If you have a more general question about a feature, feel free to ask the team in the [Zarf Kubernetes Slack Channel](https://kubernetes.slack.com/archives/C03B6BJAUJ3).
 
-We also accept contributions from the community (regardless of where a particular bug or feature is on the roadmap), so feel free to read our [contributing guidelines](./12-contribute-to-zarf/1-contributor-guide.md) and [submit a PR](https://github.com/defenseunicorns/zarf/pulls)!  You can also ask any development related questions in the [Zarf Dev Kubernetes Slack Channel](https://kubernetes.slack.com/archives/C03BP9Z3CMA).
+We also accept contributions from the community (regardless of where a particular bug or feature is in the queue), so feel free to read our [contributing guidelines](./12-contribute-to-zarf/1-contributor-guide.md) and [submit a PR](https://github.com/defenseunicorns/zarf/pulls)!  You can also ask any development related questions in the [Zarf Dev Kubernetes Slack Channel](https://kubernetes.slack.com/archives/C03BP9Z3CMA).
+
+## 2024 General Roadmap
+
+### Q1: Community Building and Refactoring
+
+- [X] - Establish a [monthly community meetup](https://github.com/defenseunicorns/zarf/issues/2202) to engage members of the community and answer questions.
+- [ ] - Refactor and add tests to library code shared with [UDS-CLI](https://github.com/defenseunicorns/uds-cli) and split into a new GitHub repository.
+- [ ] - Gather OpenSSF donation requirements and clear off pre-reqs (additional maintainers and sponsor working group).
+
+### Q2 Consistency, Docs and Donation to OpenSSF
+
+- [ ] - Consolidate and improve consistency around features such as Zarf `variables` and component `required` schema.
+- [ ] - Move [docs website](https://docs.zarf.dev) from Docusaurus to Hugo to improve maintainability going forward.
```

Comment:
Do we want to leave Hugo here?

Based on the work happening in https://github.com/defenseunicorns/zarf/pull/2315, would we rather leave it open ended to say something like:

```suggestion
- [ ] - Move [docs website](https://docs.zarf.dev) from Docusaurus to a different framework to improve maintainability going forward.
```

We can always update the reference later once the docs website is migrated as well, so I won't hold this PR up because of this.